compliance/collection/main.go

```diff
@@ -252,18 +254,28 @@ func initializeStream(ctx context.Context, cli sensor.ComplianceServiceClient) (
 
 func main() {
 	log.Infof("Running StackRox Version: %s", version.GetMainVersion())
+	clientconn.SetUserAgent(clientconn.Compliance)
 
 	// Set the random seed based on the current time.
 	rand.Seed(time.Now().UnixNano())
 
+	var nodeInventoryClient scannerV1.NodeInventoryServiceClient
+
 	if features.RHCOSNodeScanning.Enabled() {
 		// Start the prometheus metrics server
 		metrics.NewDefaultHTTPServer().RunForever()
 		metrics.GatherThrottleMetricsForever(metrics.ComplianceSubsystem.String())
-	}
 
-	clientconn.SetUserAgent(clientconn.Compliance)
+		// Set up Compliance <-> NodeInventory connection
+		niConn, err := clientconn.AuthenticatedGRPCConnection(env.NodeScanningEndpoint.Setting(), mtls.Subject{}, clientconn.UseInsecureNoTLS(true))
+		if err != nil {
+			log.Errorf("Could not initialize connection to NodeInventory service. Node Scanning will be unavailable. Error: %v", err)
```

Comment:
```suggestion
			log.Errorf("Could not initialize connection to NodeInventory service. Node Scanning will be unavailable: %v", err)
```

Nit: wrapped errors are formatted as `parent error: child error` without any (Error/Warn/Info) keywords. Makes it more idiomatic to log like IMO.